client/src/main/java/org/dst/client/cmd/DictHandler.java

```diff
@@ -21,72 +21,79 @@ public DictHandler(DefaultDstClient client) {
   public ClientResult getCmdResult(String[] cmd) {
     String result;
 
-    if ("put".equals(cmd[0])) {
-      try {
-        Map<String, String> map = new HashMap<>();
-        for (int i = 2; i < cmd.length; i = i + 2) {
-          map.put(cmd[i], cmd[i + 1]);
+    switch (cmd[0]) {
+      case "put":
+        try {
+          Map<String, String> map = new HashMap<>();
+          for (int i = 2; i < cmd.length; i = i + 2) {
+            map.put(cmd[i], cmd[i + 1]);
+          }
+          client.dicts().put(cmd[1], map);
```

Comment:
Will this line throw exception? If yes, we should catch and set to result right?

Reply:
Dnoe.